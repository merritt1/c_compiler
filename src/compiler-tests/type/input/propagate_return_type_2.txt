int b;
signed char *(a(void)) {
return 0;
}
signed int (main(void)) {
b + a();
}

