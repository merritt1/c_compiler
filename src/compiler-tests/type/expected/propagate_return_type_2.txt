=============== PARSE TREE ===============
signed int (b);
signed char (*(a(void))) {
return 0;
}
signed int (main(void)) {
(b+(a()));
}
================= SYMBOLS ================
Symbol Table ID: 0
  b -> signed int
  a -> function(pointer(signed char), [void])
  main -> function(signed int, [void])

